lib/packmap.js

```diff
@@ -9,6 +9,23 @@ module.exports = async function packmap(opts) {
   const packageJsonPath = path.resolve(process.cwd(), opts.package);
   const packageJson = require(packageJsonPath);
   const importMap = await traversePackage(packageJson);
+
+  if (opts.overrideMap) {
+    const overridingMapPath = path.resolve(process.cwd(), opts.overrideMap);
+    if (await fs.pathExists(overridingMapPath)) {
+      const overridingMap = require(overridingMapPath);
+      importMap.imports = Object.assign(
+        importMap.imports,
+        overridingMap.imports
+      );
+      console.log(`Generated import-map overridden by "${opts.overrideMap}"`);
+    } else {
+      throw new Error(
+        `Unable to find override import-map file at "${opts.overrideMap}"`
```

Comment:
This error message would be more useful if it showed the full file path that was used (`overridingMapPath` instead of `opts.overrideMap`). Reason is that it might not be clear to the user of packmap that `process.cwd()` is being used to resolve relative paths such as `./import-map.json`.

Reply:
Yup. Changed.